src/ImageProcessor.Playground/Program.cs

```diff
@@ -35,48 +33,12 @@ public class Program
         [SuppressMessage("ReSharper", "AssignNullToNotNullAttribute", Justification = "It works ok?")]
         public static void Main(string[] args)
         {
-            const string InputImages = @"..\..\..\ImageProcessor.UnitTests\Images\";
-            const string OutputImages = @"..\..\images\output";
-
-            string root = new Uri(System.Reflection.Assembly.GetExecutingAssembly().CodeBase).LocalPath;
-            string inPath = Path.GetFullPath(Path.Combine(Path.GetDirectoryName(root), InputImages));
-            string outPath = Path.GetFullPath(Path.Combine(Path.GetDirectoryName(root), OutputImages));
-
-            DirectoryInfo di = new DirectoryInfo(inPath);
-            IEnumerable<FileInfo> files = GetFilesByExtensions(di, ".jpg", ".jpeg", ".jfif", ".gif", ".bmp", ".png", ".tif");
-
-            foreach (FileInfo fileInfo in files)
-            {
-                // Start timing.
-                byte[] photoBytes = File.ReadAllBytes(fileInfo.FullName);
-                Console.WriteLine("Processing: " + fileInfo.Name);
-
-                Stopwatch stopwatch = new Stopwatch();
-                stopwatch.Start();
-
-                using (MemoryStream inStream = new MemoryStream(photoBytes))
-                using (ImageFactory imageFactory = new ImageFactory(true, true))
-                {
-                    Size size = new Size(200, 200);
-
-                    ResizeLayer layer = new ResizeLayer(size);
-
-                    imageFactory.Load(inStream)
-                                .Resize(layer)
-                                .Resolution(400, 400)
-                                .Save(Path.GetFullPath(Path.Combine(outPath, fileInfo.Name)));
-
-                    stopwatch.Stop();
-                }
-
-                // Report back.
-                long peakWorkingSet64 = Process.GetCurrentProcess().PeakWorkingSet64;
-                float mB = peakWorkingSet64 / (float)1024 / 1024;
-
-                Console.WriteLine(@"Completed {0} in {1:s\.fff} secs {2}Peak memory usage was {3} bytes or {4} Mb.", fileInfo.Name, stopwatch.Elapsed, Environment.NewLine, peakWorkingSet64.ToString("#,#"), mB);
-            }
-
-            Console.ReadLine();
+            var files = GetFilesByExtensions(new DirectoryInfo(Environment.CurrentDirectory).GetDirectories("Images").Single(), ".png");
```

Comment:
Can we make sure the playground code is not altered on merge? The timing and memory diagnostics code is very useful here.